src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -820,6 +820,7 @@ def ONNXConstantOfShapeOp:ONNX_Op<"ConstantOfShape",
       return resultTypes;
     }
   }];
+  let verifier = [{ return ::verify(*this); }];
```

Comment:
This version of verifier is going to be removed soon. Please use `let hasVerifier = 1` instead.